src/sidebar/components/ShareLinks.js

```diff
@@ -23,16 +32,12 @@ function ShareLink({ label, iconName, uri, onClick }) {
   );
 }
 
-ShareLink.propTypes = {
-  /** The name of the SVG icon to use for this link */
-  iconName: propTypes.string.isRequired,
-  /** Accessible label / tooltip for link. */
-  label: propTypes.string.isRequired,
-  /** URI for sharing this annotation on the given social service */
-  uri: propTypes.string.isRequired,
-  /** click callback (for analytics tracking) */
-  onClick: propTypes.func.isRequired,
-};
+/**
+ * @typedef ShareLinksProps
+ * @prop {object} analytics
```

Comment:
We use `{Object}` a lot more than `{object}`, it might be nice at some point to get em all the same.

Reply:
We discussed on Slack, but `object` is more specific (a non-primitive) than `Object` (which is an alias for `any`) and so `object` is preferable where applicable in new code.